dataprofiler/profilers/float_column_profile.py

```diff
@@ -332,18 +333,22 @@ def _is_each_row_float(cls, df_series: pd.Series) -> list[bool] | pd.Series[bool
         For column [1.0, np.NaN, 1.0] returns [True, True, True]
         For column [1.0, "a", "b"] returns [True, False, False]
         :param df_series: series of values to evaluate
-        :type df_series: pandas.core.series.Series
+        :type df_series: polars.series.series.Series
         :return: is_float_col
-        :rtype: Union[List[bool], pandas.Series[bool]]
+        :rtype: pl.Series
         """
         if len(df_series) == 0:
-            return list()
-        return df_series.map(NumericStatsMixin.is_float).astype("bool")
+            return pl.Series()
+        if sum(df_series.is_null()) == len(df_series):
+            return df_series
+        df_series = df_series.map_elements(NumericStatsMixin.is_float)
+        df_series = df_series.cast(bool)
+        return df_series
```

Comment:
```suggestion
        return df_series.map_elements(NumericStatsMixin.is_float).cast(bool)
```

Reply:
mypy gets update with it in that form for whatever reason

Reply:
Sorry typoed it meant upset.

Reply:
mypy doesn't like sequential together... so have to leave separate for now